src/new-components/teaser/teaser.tsx

```diff
@@ -0,0 +1,166 @@
+`;
+
+const StyledIllustration = styled(Illustration)<{ hover: boolean }>`
+  margin-bottom: 24px;
+  ${(props) =>
+    props.hover &&
+    css`
+      svg path {
+        fill: ${theme.palette.text.topline};
+      }
+    `}
+`;
+
+const ImageContainer = styled.div<{ hover: boolean }>`
+  overflow: hidden;
+  margin-bottom: 16px;
+
+  transition: transform 0.2s;
+  ${(props) => props.hover && 'transform: scale(1.05)'};
+`;
+
+const ToplineContainer = styled.div`
+  display: flex;
+  margin-bottom: 8px;
+  width: 100%;
+  justify-content: space-between;
+`;
+
+const Topline = styled.p`
+  ${TextStyles.toplineS};
+  color: ${theme.palette.text.topline};
+  margin: 0;
+`;
+
+const Timestamp = styled.p<{ hover: boolean }>`
+  ${TextStyles.timestamp};
+  color: ${(props) =>
+    props.hover ? theme.palette.text.topline : theme.palette.text.timestamp};
+  margin: 0;
+`;
+
+const TeaserText = styled.div`
+  ${TextStyles.textR};
+  margin: 0;
+`;
+
+const StyledTeaserTitle = styled.p<{
+  large: boolean;
+  hasTopline: boolean;
+}>`
+  ${TextStyles.headlineS}
+  ${(props) => props.large && TextStyles.headlineM};
+
+  margin-top: 0;
+  margin-bottom: ${(props) => (props.hasTopline ? '8px' : '16px')};
+`;
+
+const StyledIcon = styled(Icon)`
+  margin-top: 14px;
+`;
+
+export interface TeaserProps {
+  title: string;
+  topline?: string;
+  dateFormatted?: string | null;
+  teaserImage?: JSX.Element;
+  illustration?: IllustrationType;
+  linkTo?: string;
+  className?: string;
+  children: React.ReactNode | React.ReactNode[];
+}
+
+/**
+ * Teasers are used to link to another page and give the user a brief overview of the content of that page.
+ * This requires a headline, the path to the page (linkTo) and a short text, which is entered as a child.
+ * In addition, an illustration or an image, a formatted date and a topline can be displayed.
+ */
+
+const ConditionalLink = ({ to, children, onHover }) => {
+  if (to) {
+    return (
+      <Link
+        to={to}
+        onMouseOver={() => {
+          onHover(true);
+        }}
+        onMouseLeave={() => onHover(false)}
+      >
+        {children}
+      </Link>
+    );
+  }
+
+  return children;
+};
+
+export const Teaser = ({
+  topline,
+  title,
+  dateFormatted,
+  teaserImage,
+  linkTo,
+  illustration,
+  className,
+  children,
+}: TeaserProps): JSX.Element => {
+  const [hoverActive, setIsHoverActive] = useState<boolean>(false);
+  const hasToplineContainer = Boolean(topline || dateFormatted);
+
+  return (
+    <TeaserContainer className={className} hover={hoverActive}>
+      <ConditionalLink
+        onHover={(hasHover) => setIsHoverActive(hasHover)}
+        to={linkTo}
+      >
+        {teaserImage && (
+          <ImageContainer hover={hoverActive}>{teaserImage}</ImageContainer>
+        )}
+        {illustration && (
+          <StyledIllustration
+            show={illustration}
+            size={IllustrationSize.MEDIUM}
+            hover={hoverActive}
+          />
+        )}
```

Comment:
Do we want some logic here to prevent someone from adding an image as well as an illustration? Maybe a simple `{!teaserImage && illustration && (...)}` for the second condition (or the other way around, like @georgiee suggested)?

Reply:
We are responsible developers. I think a convention as mentioned is enough, isn't it ?

Reply:
We certainly are, you are right. However, if we write something like "If you pass in an image and illustration the illustration will be taken." we also need to back that with logic and as far as I can tell it doesn't exist yet, does it?

Reply:
> You can't pass in an image and illustration. If you pass in an image and illustration the illustration will be taken.

I added @fischeversenker's logic to make the component fit the description